scripts/tui-pty-smoke.py

```
@@ -13,6 +13,47 @@
 import time
 
 
+def exec_or_exit(binary: str, argv: list[str], env: dict[str, str]) -> None:
+    try:
+        os.execve(binary, argv, env)
```

Ruff (0.16.3)

[error] Starting a process without a shell

(S606)